### ISSUE TITLE
ctc dex chart always returns could not determine dex class of pool

### ISSUE THREAD

**Issue by Jacks349** [fei-protocol/checkthechain#36]
Hello, first of all thank you for working on this tool because it's truly awesome. I have a little problem when i try to get the chart of a given pool. Whenever i try to get the chart i always get the following error: `could not determine dex class of pool`. Here is the command i tried to use: ctc dex chart 0x5c6Ee304399DBdB9C8Ef030aB642B10820DB8F56
I tried the same thing both from a Windows 10 machine using an Infura mainnet node and a Linux machine with a self hosted node, both using Geth.
It looks like the line `dex_pool = await db.async_query_dex_pool(address=pool, network=network)` in line 120 of **dex_class_utils.py** always returns `None`. Am i doing something wrong?

**Comment by kurtsreynolds** [comment]
Maybe a similar issue to what I am seeing - 0 pools found when running `ctc dex pools`

Windows:
![image](https://user-images.githubusercontent.com/114658317/193005526-42e2d7e5-90cd-44e7-9bb4-ee70d7975ef1.png)

Linux:
![image](https://user-images.githubusercontent.com/114658317/193005672-d5763f03-df2c-4991-9b90-2392b5afb161.png)

**Comment by Jacks349** [comment]
> Maybe a similar issue to what I am seeing - 0 pools found when running `ctc dex pools`
> 
> Windows: ![image](https://user-images.githubusercontent.com/114658317/193005526-42e2d7e5-90cd-44e7-9bb4-ee70d7975ef1.png)
> 
> Linux: ![image](https://user-images.githubusercontent.com/114658317/193005672-d5763f03-df2c-4991-9b90-2392b5afb161.png)

I get the exact same output with ctc dex pools

**Comment by kurtsreynolds** [comment]
Waiting for the command to run/do anything - has been 1 min so far. Will report back if/when it runs. 👍 

ctc cg --update worked fine btw.

**Comment by sslivkoff** [comment]
what rpc provider are you using? you may need a beefy node to index these things or else it can take a long time

**Comment by Jacks349** [comment]
Indeed, i didn't run `ctc dex pools --update`. That was the problem. Thanks a lot @sslivkoff !

**Comment by sslivkoff** [comment]
to resolve this issue should
1) add info in cli help message
2) when dex pool commands run with no existing index, should either auto create the index or simply display a message telling user to create index
